clients/cobol-dialect-api/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import * as vscode from "vscode";
+
+interface DialectV1 {
+  apiVersion: 1;
+  name: string;
+  description: string;
+  jar: vscode.Uri;
+  snippets: vscode.Uri;
+}
+const mainExtensionId = "BroadcomMFD.cobol-language-support";
+
+interface unregister {
+  (): void;
+}
+
+export const registerDialect = async (
```

Comment:
I think it makes sense to add a simple test for this.

Reply:
I will do this in next iteration that will include refactoring of the API exported by COBOL LS extension. The focus of this PR is on the clean up of the API faced by dialect extender.